lib/strings.js

```diff
@@ -88,45 +86,41 @@ const subst = (
   return result;
 };
 
-const fileExt = (
-  // Extract file extension in lower case with no dot
-  fileName // string, file name
-  // Returns: string
-  // Example: fileExt('/dir/file.txt')
-  // Result: 'txt'
-) => (
+// Extract file extension in lower case with no dot
+//   fileName - <string>, file name
+// Returns: <string>
+// Example: fileExt('/dir/file.txt')
+// Result: 'txt'
```

Comment:
'txt'?
maybe `Result: <string>, file extension without dot`?

Reply:
@lundibundi, `'txt'` here is the result of running the example, the text you are proposing must be above the example in 'Returns:', describing what to expect in general as a result of calling this function.

Reply:
Yeah, I have made a note about this syntax but didn't update the comment.
https://github.com/metarhia/common/pull/194#discussion_r226683471